openmmtools/integrators.py

```diff
@@ -999,31 +999,38 @@ class LangevinSplittingIntegrator(ThermostatedIntegrator):
         - g-BAOAB with solvent-solute splitting, K_r=K_p=2:
             splitting="V0 V1 R R O R R V1 R R O R R V1 V0"
 
+    Attributes
+    ----------
+    _kinetic_energy : str
+        This is 0.5*m*v*v by default, and is the expression used for the kinetic energy
+
     References
     ----------
     [Leimkuhler and Matthews, 2015] Molecular dynamics: with deterministic and stochastic numerical methods, Chapter 7
     """
 
+    _kinetic_energy = "0.5 * m * v * v"
+
     def __init__(self,
                  splitting="V R O R V",
```

Comment:
@jchodera : Is there are reason to prefer this here? A single `O` solve of length dt will have the same effect (in distribution) as two consecutive `O` solves of length dt/2 -- `O O` just takes 2x as many gaussian r.v.'s as `O`...